### DISCUSSION TITLE
Access to FBA Grade and Resell report Data via SP-API

### DISCUSSION THREAD

**Discussion by Jean-PaulChampy** [amzn/selling-partner-api-models#4200] [Q&A] [answered]
Hello everyone,

I hope you're all doing well.

Grade and Resell is a crucial part of what can happen to a product in our inventory, and it significantly impacts our revenue. Understanding how different products are affected by the Grade and Resell process can provide valuable insights into our business performance and help us optimize our daily operations.

Currently, my team is looking to collect detailed data on products that go through Grade and Resell. Having access to this data through a report like the one available in Seller Central would allow us to:

Analyze how various products are impacted by the Grade and Resell process.
Make informed decisions to adjust pricing, inventory management, and sales strategies.
Develop and implement daily operational plans based on real-time data.
We have explored the existing SP-API reports, such as Customer Returns, Inventory Health, and Removal Orders, but haven't been able to recreate the comprehensive view provided by the Grade and Resell report in Seller Central.

Is there a way to access detailed Grade and Resell data through the SP-API, or does an endpoint or report exist that we may have overlooked?

Having access to this data is essential for us to fully understand the impact of the Grade and Resell process on our revenue and adjust our daily operations accordingly.

Thank you for your support and guidance!

Best regards,
Jean-Paul

**Comment by mafge** [comment, marked as answer]
Hi @Jean-PaulChampy,
thanks for reaching out!
Unfortunately there is no comparable report available in the API. As you already mentioned, you could try to use the existing reports to build a dataset that is similar to the one you are looking for. Our [FBA reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-fba) could be helpful for that.
Best,
Marc
Selling Partner Developer Services
